O/OpenFHE/common.jl

```diff
@@ -0,0 +1,75 @@
+using BinaryBuilder, Pkg
+
+function prepare_openfhe_build(name::String)
+    # Collection of sources required to complete build
+    sources = [
+        GitSource("https://github.com/openfheorg/openfhe-development.git",
+                  "7b8346f4eac27121543e36c17237b919e03ec058"),
```

Comment:
IMHO the hash needs to be an input variable as well, since it changes with each version. You could, e.g., add a second argument `git_hash` to `prepare_openfhe_build`